tools/refract-filter.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import argparse
+import sys
+import yaml
```

Comment:
Good point, you don't necessarily need yaml and therefore happily live with the standard lib.